Workbooks/Virtual Machines/Availability/settings.json

```diff
@@ -0,0 +1,8 @@
+{
+    "$schema": "https://github.com/Microsoft/Application-Insights-Workbooks/blob/master/schema/settings.json",
+    "name":"Availability",
+    "author": "Microsoft",
```

Comment:
this file no longer needs to exist (and it should probably even give you validation/build errors if it does?)

Reply:
how can I fix this?